korangar/src/graphics/passes/point_shadow/shader/entity.wgsl

```diff
@@ -6,9 +6,17 @@ struct PassUniforms {
 
 struct InstanceData {
     world: mat4x4<f32>,
+    texture_top_left: vec2<f32>,
+    texture_bottom_left: vec2<f32>,
+    texture_top_right: vec2<f32>,
+    texture_bottom_right: vec2<f32>,
     texture_position: vec2<f32>,
     texture_size: vec2<f32>,
     depth_offset: f32,
+    depth_extra: f32,
+    angle: f32,
+    foo: f32,
```

Comment:
Dito as in the bindless point shader file. I won't repeat this for the other shaders.

Reply:
Removed the extra parameters